pom.xml

```
@@ -324,7 +324,7 @@
             <phase>package</phase>
             <goals><goal>generate</goal></goals>
             <configuration>
-              <inputSpec>${project.build.directory}/generated/swagger/convertedOutput.json</inputSpec>
+              <inputSpec>${project.build.directory}/generated/swagger/admin/swagger.json</inputSpec>
```

This and other `inputSpec` seem to be flipped, so
```suggestion
              <inputSpec>${project.build.directory}/generated/swagger/public/swagger.json</inputSpec>
```

---

on line 323 the id, is 
```            generate-html-admin
```
I'm assuming @shabd67 is just trying to make it consisent, (so that 'generate-html-admin') actually does generate the admin html.

---

Thanks! That line was just outside the default diff view, so I didn't have that point of reference.